configuration/management/commands/refresh_views.py

```
@@ -0,0 +1,28 @@
+    "data_current_benefits",
+    "data_householdmembers",
+    "data_immediate_needs",
+    "data_previous_benefits",
+    "data_programs",
+]
+
+
+class Command(BaseCommand):
+    help = "Refresh all materialized views"
+
+    def handle(self, *args, **kwargs):
```

Ruff (0.12.2)

Unused method argument: `args`

(ARG002)